docker/sia/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM golang AS builder
+LABEL maintainer="NebulousLabs <devs@nebulous.tech>"
+
+ENV GOOS linux
+ENV GOARCH amd64
+
+RUN git clone https://gitlab.com/NebulousLabs/Sia.git && \
+    cd Sia && \
+    git checkout siad-v150-pre-rc && \
+    make release
```

Comment:
This whole file should be gone - we should tag the RC, publish an image and use that one.